acceptance/acceptance_test.go

```diff
@@ -256,6 +257,11 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		}
 	}
 
+	id := uuid.New()
+	uniqueName := strings.Trim(base32.StdEncoding.EncodeToString(id[:]), "=")
+	require.Greater(t, len(uniqueName), 8, uniqueName)
```

Comment:
Could it be less than 8 characters?

Reply:
No, it should never be (128 bits with 5 bits per char). I've added this as a sanity check, maybe not eve necessary.

Reply:
Removed a3327908246bc24dbd7fcbc1b4b91433c254c53a